dashboard/app/views/datasets/show_manifest.html.haml

```diff
@@ -0,0 +1,9 @@
+- require 'json'
```

Comment:
this line should be able to be deleted (now that we moved the JSON parsing from HAML to React)